<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>otp</class>
 <widget class="QDialog" name="otp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OTP Verification</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(25, 31, 50);
</string>
  </property>
  <widget class="QLabel" name="enter_text">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 11pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Enter the code from the email we sent to</string>
   </property>
  </widget>
  <widget class="QLabel" name="wardenfort">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>80</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/admin/Downloads/wardenfort_word (1).png</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>67</y>
     <width>529</width>
     <height>461</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(44, 60, 75, 0.6); 
border-radius: 50px;</string>
   </property>
   <widget class="QLabel" name="email_show">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>152</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(1, 253, 245);
background-color: rgba(255, 255, 255, 0);
font: 700 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>null</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resendButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>327</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);
text-decoration: underline;
font: 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Resend</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="timer">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>239</y>
     <width>51</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 700 12pt &quot;Segoe UI&quot;;
color: rgb(1, 253, 245);
</string>
   </property>
   <property name="text">
    <string>null</string>
   </property>
  </widget>
  <widget class="QLabel" name="code_text">
   <property name="geometry">
    <rect>
     <x>248</x>
     <y>360</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Didnâ€™t receive the code? </string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>470</y>
     <width>275</width>
     <height>34</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>275</width>
     <height>34</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>275</width>
     <height>34</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-radius: 15px; 
background-color: rgb(22, 27, 46);
color: rgb(1,253,245);
}
QPushButton:hover {
}
QPushButton:pressed {
background-color: rgb(24, 40, 55); /* Adjust pressed color if needed */
}</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="cyber_bg">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>150</y>
     <width>421</width>
     <height>331</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/admin/Downloads/cyber (1).png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="verify_text">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>VERIFY</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
border-radius: 15px; 
background-color: rgb(44, 60, 75);
color: rgb(1, 253, 245);
	font: 700 18pt &quot;Segoe UI&quot;;
    padding-left: 20px; padding-top: 6px;
}
</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly|Qt::ImhSensitiveData</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="text2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
border-radius: 15px; 
background-color: rgb(44, 60, 75);
color: rgb(1, 253, 245);
	font: 700 18pt &quot;Segoe UI&quot;;
    padding-left: 20px; padding-top: 6px;
}
</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly|Qt::ImhSensitiveData</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="text3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>310</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
border-radius: 15px; 
background-color: rgb(44, 60, 75);
color: rgb(1, 253, 245);
	font: 700 18pt &quot;Segoe UI&quot;;
    padding-left: 20px; padding-top: 6px;
}
</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>310</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
border-radius: 15px; 
background-color: rgb(44, 60, 75);
color: rgb(1, 253, 245);
	font: 700 18pt &quot;Segoe UI&quot;;
    padding-left: 20px; padding-top: 6px;
}
</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text6">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>310</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
border-radius: 15px; 
background-color: rgb(44, 60, 75);
color: rgb(1, 253, 245);
	font: 700 18pt &quot;Segoe UI&quot;;
    padding-left: 20px; padding-top: 6px;
}
</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>310</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
border-radius: 15px; 
background-color: rgb(44, 60, 75);
color: rgb(1, 253, 245);
	font: 700 18pt &quot;Segoe UI&quot;;
    padding-left: 20px; padding-top: 6px;
}
</string>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>submitButton</zorder>
  <zorder>enter_text</zorder>
  <zorder>verify_text</zorder>
  <zorder>wardenfort</zorder>
  <zorder>cyber_bg</zorder>
  <zorder>code_text</zorder>
  <zorder>timer</zorder>
  <zorder>text1</zorder>
  <zorder>text2</zorder>
  <zorder>text3</zorder>
  <zorder>text5</zorder>
  <zorder>text6</zorder>
  <zorder>text4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
